assets/js/components/notifications/ErrorNotifications.test.js

```diff
@@ -109,7 +109,7 @@ describe( 'ErrorNotifications', () => {
 			viewContext: VIEW_CONTEXT_MAIN_DASHBOARD,
 		} );
 
-		await act( waitForRegistry );
```

Comment:
This is already wrapped from the return of `render` and I don't think `act` can be nested like this.